hibernate-types-52/src/test/java/com/vladmihalcea/hibernate/util/providers/DataSourceProvider.java

```diff
@@ -12,20 +12,5 @@
 
     DataSource dataSource();
 
```

Comment:
I wouldn't delete those methods as they could be useful. These are JDBC client-specific settings, so they should not interfere with this feature.